criu/pagemap-cache.c

```diff
@@ -154,10 +154,16 @@ static int pmc_fill_cache(pmc_t *pmc, const struct vma_area *vma)
 	BUG_ON(pmc->map_len < size_map);
 	BUG_ON(pmc->fd < 0);
 
-	if (pread(pmc->fd, pmc->map, size_map, PAGEMAP_PFN_OFF(pmc->start)) != size_map) {
-		pmc_zap(pmc);
-		pr_perror("Can't read %d's pagemap file", pmc->pid);
-		return -1;
+	size_t have_read = 0;
+	while (have_read < size_map) {
+		ssize_t ret = pread(pmc->fd, (char *)pmc->map + have_read, size_map - have_read,
+		                    PAGEMAP_PFN_OFF(pmc->start) + have_read);
+		if (ret == -1) {
```

Comment:
CRIU uses `< 0` to check syscall return value, please keep this consistent

Reply:
do we need to report an error if ret is 0 here?